### ISSUE TITLE
A11y_GradeB_VSCode_Unable to find Rating - Blind:  Tab navigation to Rating link in the extension its name as "rating" and the visual rating is not read

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#182487]
**GitHubTags**:#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#DesktopApp;#BM-VisualStudioCode-Win32-Nov2021;#JAWS;#Win32;#Win11;#AINotInScope;#External;#External:JAWS;#ThirdParty;

## Environment Details:​
Application Name: Visual Studio Code
Version: 1.79.0-insider (user setup)
OS build	25300.1000
JAWS 2023.2303.144

## Repro Steps:

1. Launch Visual Studio Code insiders
2. Navigate to extension and search for "Javascript"
3. start JAWS
4. tab to the rating section at the top 

## Actual:
Tab navigation to Rating link in the extension its name as "rating" and the visual rating is not read, It reads left right parenthesis the number and control as link
and in speech viewer we can see the text read 

## Expected:
Tab navigation to Rating link in the extension it should be read as average ration 4 of out 5 link 540

##User Impact:
screen reader users will not know the rating of the extension 

## Attachment
[Unable to find rating-JAWS.webm](https://github.com/microsoft/vscode/assets/24580570/14d05f11-cd09-4840-a0bc-224ce6d09947)

**Comment by sandy081** [comment]
@meganrogge Running through you first to see if you have any insights.

**Comment by meganrogge** [comment]
We provide the extension's rating in the aria label when a user is navigating the extension view. IMO, this is more convenient and accessible than providing it via tab navigation in the extension's page.

**Comment by meganrogge** [comment]
In testing, I can see that this is only working for some extensions and not sure why that would be @sandy081. Any thoughts?

**Comment by meganrogge** [comment]
In testing, I can see that the correct aria label - with rating info, is only applied to some of the extensions. not sure why that would be - any thoughts @sandy081 ?

**Comment by meganrogge** [comment]
https://github.com/microsoft/vscode/pull/181897/files

**Comment by meganrogge** [comment]
Actually, it's now being applied to all. But I did see a time when that was not the case. We should also be rounding this to a reasonable decimal 


https://github.com/microsoft/vscode/assets/29464607/6d8f3511-eaa6-4f69-948a-027216cc4c64

**Comment by meganrogge** [comment]
This also works for me on Windows with NVDA, so I believe it could be a JAWS specific issue.

**Comment by fsteffi** [comment]
hi @meganrogge  the issue does not repro with NVDA, its does only with JAWS

**Comment by fsteffi** [comment]
hi @meganrogge  could you please keep this bug open as per external bug process we need to log a bug to NVDA team and have the bug in open state

**Comment by meganrogge** [comment]
Done, thanks. And I believe you mean the Orca team

**Comment by sbatten** [comment]
As this bug is upstream, if we are leaving it open, can we please lower the severity as this makes us red in S360

**Comment by isidorn** [comment]
@sbatten good point. I reached out to the accessibility team for guidance.

**Comment by fsteffi** [comment]
Closing this bug as per mail guidance RE: Ask - Log bugs from VS Code Usability Study 
Hi Steffi,

I checked on this with Roberto and received confirmation that it can be closed.

[A11y_GradeB_VSCode_Unable to find Rating - Blind: Tab navigation to Rating link in the extension its name as "rating" and the visual rating is not read · Issue #182487 · microsoft/vscode · GitHub](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fvscode%2Fissues%2F182487&data=05%7C01%7Cv-stfe%40microsoft.com%7Cf00873756ff04bb9ed3408db8c8bbddb%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638258304100313803%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3bpV2IT%2FQmKRhZxqfXbJZB19uB1cN1dvqolfch8IBdk%3D&reserved=0)

Thanks,
Tanaz

#Closed;
